test/models/album_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class AlbumTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+
+  # validates :artist, presence: true
+  # validates :name, presence: true
+
+  test 'album with no name is invalid' do
+    assert albums(:one).valid?
```

Comment:
Excellent test! However, it is failing from this assertion due to the unique name validation... because our fixtures are all using the same value ("MyString") for `name` and `artist`.